### PR TITLE
VersionId Support in removeObjects api

### PR DESCRIPTION
**Pull request by prakashsvmx**
VersionId Support in removeObjects api

### CODE REVIEW

**Review by kannappanr — APPROVED** [review]
LGTM

File: examples/remove-objects.js [hunk 1]
```diff
@@ -18,27 +18,27 @@
...
   useSSL: false,
   accessKey: 'YOUR-ACCESSKEYID',
   secretKey: 'YOUR-SECRETACCESSKEY'
-});
+})
 
 var objectsList = []
 
 // List all object paths in bucket my-bucketname.
 var objectsStream = s3Client.listObjects('my-bucketname', '', true)
```
Comment (ebozduman) [inline examples/remove-objects.js:34]: Assuming the other 2 suggested changes/comments are accepted, this example script can be simplified to work for both versioned and non-versioned scenarios:
```
var objectsList = []
var bucket = '<bucket-name>'
var prefix = '<prefix-name>'
var recursive = <true | false>
var includeVersion = <true | false>

// List all object paths in bucket
var objectsStream = s3Client.listObjects(bucket, prefix, recursive, {IncludeVersion: includeVersion})

objectsStream.on('data', function(obj) {
  if (includeVersion) {
    objectsList.push(obj)
  } else {
    objectsList.push(obj.name)
  }
})

objectsStream.on('error', function(e) {
  return console.log(e)
})

objectsStream.on('end', function() {
  s3Client.removeObjects(bucket, objectsList, function(e) {
    if (e) {
      return console.log(e)
    }
    console.log("Success")
  })
})
```

**Review by ebozduman — CHANGES_REQUESTED** [review]

File: examples/remove-objects.js (hunk 1 above)
Comment (prakashsvmx) [reply examples/remove-objects.js:34]: Thank you for the suggestion. I will refactor.

**Review by ebozduman — CHANGES_REQUESTED** [review]
Couple of minor changes...

**Review by ebozduman — APPROVED** [review]
LGTM

**Review by kanagarajkm — APPROVED** [review]

**Pull request merged**